editors/neovim/Portfile

```
@@ -48,7 +48,7 @@ cmake.build_type        Release
 configure.args-append   -DLUA_PRG=${prefix}/bin/luajit
 
 # Building parsers is normally an extra step, see https://github.com/neovim/neovim/issues/29042
-patchfiles              embed-parsers-build.diff
+patchfiles              embed-parsers-build.diff  patch-nlua0-includes.diff
```

Usually we put each patch file on their own line, like so:

```suggestion
patchfiles              embed-parsers-build.diff \
                        patch-nlua0-includes.diff
```